nise/manifest.py

```diff
@@ -69,10 +69,18 @@ def generate_manifest(fake, template_data):
     range_str = _manifest_datetime_range(bp_start, bp_end)
     assembly_id = uuid4()
     report_id = fake.sha256(raw_output=False)
-    report_key = '/{report_name}/{range_str}/{assembly_id}/{report_name}-1.csv.gz'
-    report_key = report_key.format(report_name=report_name,
-                                   range_str=range_str,
-                                   assembly_id=assembly_id)
+    prefix_name = template_data.get('prefix_name')
```

Comment:
Could you change the code to: 
```
prefix_name = template_data.get('prefix_name', '')
```

and get remove the if/else?

Reply:
prefix_name is None when one isn't supplied so this would make a directory called "None".  I'm looking at alternatives to getting rid of the if/else. I agree it's ugly.

Reply:
Also, I think we need the `/` when prefix is not supplied.  This is how it looks on a real AWS bucket with no prefix configured.  With a prefix the reportKeys show up without the /